### DISCUSSION TITLE
Updated doc on attestorPlugin (ClaimPluginActor) and rules for Attestation Policy

### DISCUSSION THREAD

**Discussion by radhay-samagra** [Sunbird-RC/community#218] [Issues]
I would like to add another similar issue on top of this, for attestation policies, I have seen attestorEntity mentioned in the docs [here](https://github.com/Sunbird-RC/community/blob/main/developer-documentation/creating-your-own-schemas.md?plain=1#L82) to set which entity can attest the claim, but I have nowhere found the implementation details on attestorPlugin attribute which I found out [here](https://github.com/Sunbird-RC/demo-education-registry/blob/main/schemas/Teacher.json#L90). Similar issues with property attestationProperties. Can you point me out to a updated doc mentioning all of these attributes?

**Comment by pramodkvarma** [comment]
@tejash-jl @parthlawate please help and also see if we can put together quick docs on these. Thanks.
